test/unit/adapters/http.js

```diff
@@ -1383,13 +1426,19 @@ describe('supports http with nodejs', function () {
       // call cancel() when the request has been sent, but a response has not been received
       source.cancel('Operation has been canceled.');
     }).listen(4444, function () {
-      axios.get('http://localhost:4444/', {
-        cancelToken: source.token
-      }).catch(function (thrown) {
-        assert.ok(thrown instanceof axios.Cancel, 'Promise must be rejected with a CanceledError object');
-        assert.equal(thrown.message, 'Operation has been canceled.');
-        done();
-      });
+      assert.rejects(
```

Comment:
uh, looks like a floating promise

Reply:
nm, it wasn't since we call `.then(done).catch(done)` on it, but I added `void` before it to clarify (per convention [here](https://typescript-eslint.io/rules/no-floating-promises/))